_includes/manuals/1.18/3.1.3_puppet_versions.md

```diff
@@ -9,42 +10,14 @@ Foreman integrates with Puppet and Facter in a few places, but generally using a
     <th><a href="/manuals/{{page.version}}/index.html#3.5.5FactsandtheENC">External node classifier</a></th>
   </tr>
   <tr>
-    <td>0.25.x</td>
+    <td>0.x-2.x</td>
     <td>Not supported</td>
     <td>Untested</td>
     <td>Untested</td>
-    <td>Supported <span class='footnote'>*</span></td>
+    <td>Untested <span class='footnote'>*</span></td>
   </tr>
   <tr>
-    <td>2.6.0 - 2.6.5</td>
-    <td>Not supported</td>
-    <td>Untested</td>
-    <td>Untested</td>
-    <td>Supported <span class='footnote'>*</span></td>
-  </tr>
-  <tr>
-    <td>2.6.5+</td>
-    <td>Not supported</td>
-    <td>Supported</td>
-    <td>Supported</td>
-    <td>Supported</td>
-  </tr>
-  <tr>
-    <td>2.7.x</td>
-    <td>Not supported</td>
-    <td>Supported</td>
-    <td>Supported</td>
-    <td>Supported</td>
-  </tr>
-  <tr>
-    <td>3.0.x - 3.5.x</td>
-    <td>Not supported</td>
-    <td>Supported</td>
-    <td>Supported</td>
-    <td>Supported</td>
-  </tr>
-  <tr>
-    <td>3.6.x - 3.8.x</td>
+    <td>3.x</td>
```

Comment:
This implies we support any puppet version. I'd suggest we make this 3.8.x and mark anything < 3.8 as untested.